paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -127,19 +127,49 @@ class PaymentSheet internal constructor(
 
     @Parcelize
     data class Address(
+        /**
+         * City, district, suburb, town, or village.
+         */
         val city: String? = null,
+        /**
+         * Two-letter country code (ISO 3166-1 alpha-2).
+         */
         val country: String? = null,
+        /**
+         * Address line 1 (e.g., street, PO Box, or company name).
+         */
         val line1: String? = null,
+        /**
+         * Address line 2 (e.g., apartment, suite, unit, or building).
+         */
         val line2: String? = null,
+        /**
+         * ZIP or postal code.
+         */
         val postalCode: String? = null,
+        /**
+         * State, county, province, or region.
+         */
         val state: String? = null
     ) : Parcelable
 
     @Parcelize
     data class BillingDetails(
+        /**
+         * The customer's billing address.
+         */
         val address: Address? = null,
+        /**
+         * The customer's email.
+         */
         val email: String? = null,
+        /**
+         * The customer's full name.
```

Comment:
should we include something about spaces? I can see myself questioning if it needs to be "skyler reimer" or "skylerreimer"

ditto for the others if we care about them like we do phone number.

Reply:
Great point.  

The requirements for these fields can vary based on what the payment method provider ultimately requires unfortunately.  Maybe we change this to something like:

"The customer's full name. The value set is displayed in the payment sheet as-is. Depending on the payment method, the customer may be required to edit this value."

Reply:
@yuki-stripe I like that a lot better.